src/Integration/Resources/Strings.resx

```diff
@@ -660,4 +660,20 @@ This is the general format to use when writing errors to output window or when t
     <value>   {0}</value>
     <comment>Output window message format. Used whenever the message should be displayed as a sub-message from a step message so the user can clearly identify the grouping. {0} the actual message to display.</comment>
   </data>
+  <data name="SonarQubeProjectSolutionWideRuleSetPrefix" xml:space="preserve">
+    <value>SonarQube</value>
+    <comment>Used in combination with SonarQubeRuleSetNameFormat when the ruleset is the solution ruleset</comment>
+  </data>
+  <data name="SonarQubeProjectSpecificRuleSetPrefix" xml:space="preserve">
+    <value>Local</value>
```

Comment:
Let's use SonarQube here as well. So project specific and solution level ruleset files will have the same name.